modules/log-collector-rsyslog-server.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-rsyslog-server_{context}"]
+= Configuring the collector to listen for connections as an syslog server
+
+You can configure your log collector to collect logs from outside the cluster by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR). This enables you to use a single common log store for logs that are collected from both inside and outside of your {product-title} cluster.
```

Comment:
@alanconway @jlarriba do we need to make some statement regarding the intent of providing this input, expectations, etc.  My recollection is we expect to only receive logs which are of a journal format and they will be classified as infra.  What happens when someone tries to send us logs of a non-journal format?  Does it matter

Reply:
Is there any similar limitation for http? I'm working on those docs as well now.

Reply:
Yes, we are only supporting a very narrow use case here that goes to infra, so we are not really intending to expose this feature to general logging. It is intended to be used by OpenStack compute nodes journal logs, so I agree on making this more specific in the direction Jeff is suggesting.

The limitation for http is similar: it only admits audit logs and they always classified as audit. It is a very similar narrow use case. This receivers are def not intended as of 5.9 as broad receivers of different types of logs.

We discussed in the past that having that feature would entail a big rework of the whole operator and that's scheduled for version 6.

Reply:
Thanks so much for the information! I will update the docs accordingly.